account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
+			"identity": model,
+			"err":      err.Error(),
+		}, "Unable to create the identity")
 		return errors.WithStack(err)
```

Comment:
@kwk The problem with that error is not the wrapping itself, but that it contain little to no structural contextual information.

And it might down the line be just as valuable to know when NotFound happens. We don't know why the server is slow because we're only seeing 5 'found' requests and missing the 5000 not found requests. We need to figure out _why_ NotFound happens as well, where and why are they coming? Are we providing some wrong info in some other location.

The structual logs shouldn't just scroll by on the console log, it needs to be stored somewhere that is indexable. 

Which gives us the option to; give me everything that happened to Request X, or what has happened the last 30 days to Identity X etc etc